rewriter/Minitest.cc

```diff
@@ -653,6 +654,99 @@ ast::ExpressionPtr runSingle(core::MutableContext ctx, bool isClass, ast::Send *
             return constantMover.addConstantsToExpression(send->loc, move(method));
         }
 
+        case core::Names::its().rawId(): {
+            if (block == nullptr || !send->recv.isSelfReference() ||
+                (!insideDescribe && requiresSecondFactor(send->fun))) {
+                return nullptr;
+            }
+
+            if (send->numPosArgs() != 1) {
+                return nullptr;
+            }
+
+            auto &arg = send->getPosArg(0);
+
+            // Only handle symbol arguments for now (its(:attribute))
+            auto argLit = ast::cast_tree<ast::Literal>(arg);
+            if (argLit == nullptr || !argLit->isName()) {
+                return nullptr;
+            }
+
+            auto attributeName = argLit->asName();
+            auto argString = attributeName.show(ctx);
+
+            // Create the describe block name
+            auto describeTestName = fmt::format("<describe '{}'>", argString);
+            auto describeName =
+                ast::MK::UnresolvedConstantParts(arg.loc(), {ctx.state.enterNameConstant(describeTestName)});
+
+            auto isExpectedName = ctx.state.enterNameUTF8("is_expected");
+            auto expectName = ctx.state.enterNameUTF8("expect");
+
+            // Transform the its block body to replace is_expected with expect(subject.attribute)
+            class IsExpectedTransformer {
+            private:
+                core::NameRef attributeName;
+                core::NameRef isExpectedName;
+                core::NameRef expectName;
```

Comment:
Once you have statically registered these names, there is no benefit to passing them into the constructor. They are always bound to the well-known name. We should drop these two class fields and compare against those names directly.

e.g, `send->fun == core::Names::isExpected()` in the condition below.